pipeline-scripts/buildlib.groovy

```diff
@@ -1440,6 +1441,13 @@ def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_sig
     }
 
     def productVersion = el_major >= 8 ? "OSE-${major_minor}-RHEL-${el_major}" : "RHEL-${el_major}-OSE-${major_minor}"
+    def brewTag = "rhaos-${major_minor}-rhel-${el_major}-candidate"
+    def embargoedBrewTag = "--embargoed-brew-tag rhaos-${major_minor}-rhel-${el_major}-embargoed"
+    if (for_ironic) {
+        productVersion = "OSE-8-RHEL-${el_major}-IRONIC"
```

Comment:
I saw in the ticket the new product version name is `OSE-IRONIC-4.11-RHEL-8`. So maybe?
```suggestion
        productVersion = "OSE-IRONIC-${major_minor}-RHEL-${el_major}"
```

Reply:
good catch, thanks.